webpage/templates/source_detail.txt

```diff
@@ -6,13 +6,21 @@ Source: {{ source.common_name }}
 Basic source info
 -----------------
 
-TODO
+{% if source.source_id < 10 %} 
```

Comment:
I tried to pass the source_id to jinja as a string like 000001 but jinja automatically converts it to a int.
These if-statements work but are not nice, I know.

Reply:
This is horrible. Please learn http://jinja.pocoo.org/docs/dev/templates and use that for formatting.

Reply:
See comment above